modules/asio/1.21.0/patches/add_build_file.patch

```diff
@@ -0,0 +1,12 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,9 @@
++cc_library(
```

Comment:
Ideally this BUILD file should be upstreamed so we don't need it for future version, can you help with that?

Reply:
I can try. However, I'm not sure if the author is accepting other build systems. Let's see.

Reply:
Thanks!